### PR TITLE
Run

### PR DESCRIPTION
**Pull request by cvogt**
@rockjam wanna give this quick review :)? pickOne may be useful to you.

### CODE REVIEW

**Comment by rockjam** [comment]
@cvogt will take a look

File: stage1/Stage1Lib.scala [hunk 1]
```diff
@@ -118,6 +112,67 @@ class Stage1Lib( val logger: Logger ) extends BaseLib{
     }
   }
 
+  /** shows an interactive dialogue in the shell asking the user to pick one of many choices */
+  def pickOne[T]( msg: String, choices: Seq[T] )( show: T => String ): Option[T] = {
+    if(choices.size == 0) None else if(choices.size == 1) Some(choices.head) else {
+      Option(System.console).map{
```
Comment (rockjam) [inline stage1/Stage1Lib.scala]: Why System.console and not scala.Console?

File: stage1/Stage1Lib.scala [hunk 2]
```diff
@@ -118,6 +112,67 @@ class Stage1Lib( val logger: Logger ) extends BaseLib{
...
+  /** interactively pick one main class */
+  def runClass( mainClasses: Seq[Class[_]] ): Option[Class[_]] = {
+    pickOne( "Which one do you want to run?", mainClasses )( _.toString )
+  }
+
+  def mainClasses( targetDirectory: File, classLoader : ClassLoader ): Seq[Class[_]] = {
+    val arrayClass = classOf[Array[String]]
+    val unitClass = classOf[Unit]
+
+    listFilesRecursive(targetDirectory)
```
Comment (rockjam) [inline stage1/Stage1Lib.scala]: Indent on .filter

File: test/test.scala [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import cbt._
+package cbt
+package test
```
Comment (rockjam) [inline test/test.scala]: Which style is encouraged in cbt codebase? 
`package cbt.some.thing` 
or 
```
package cbt
package some
package thing
```

File: stage1/Stage1Lib.scala (hunk 1 above)
Comment (cvogt) [inline stage1/Stage1Lib.scala]: The io stuff in scala's standard library feels mostly ad-hoc and outdated, so I tend to use the JDK ones.

File: test/test.scala (hunk 3 above)
Comment (cvogt) [inline test/test.scala]: @rockjam it's not a style. It is semantically different.
```
package cbt
package test
```
is equivalent to 
```
package cbt.test
import cbt._
```

So my view is let's only it when we need the import equivalent

File: stage1/Stage1Lib.scala (hunk 2 above)
Comment (cvogt) [inline stage1/Stage1Lib.scala]: fixed

File: test/test.scala (hunk 3 above)
Comment (rockjam) [inline test/test.scala]: Well, that was a discovery for me!

**Pull request merged**